server/db.js

```diff
@@ -1,16 +1,17 @@
 const mysql = require('mysql');
 const password = require('./password.js');
 const connection = mysql.createConnection({
-    host: 'localhost',
-    user: 'root',
+    host: 'product-database.cdrcwxiifuzp.us-east-2.rds.amazonaws.com',
+    port: 3306,
+    user: 'admin',
     password: password,
     database: 'Products'
 });
 
 connection.connect(() => console.log('Connected to Database!'));
 
-let selectAllProducts = (callback) => {
-    connection.query('SELECT * FROM Products.info', (err, results) => {
+let updateOneProduct = (callback, id, newAvg, newTotal) => {
```

Comment:
The formatting Garrett mentioned during our group-mini-app night is pretty useful for preventing DB attacks in queries through user input.  Basically, in the query string instead of `'UPDATE products SET field=${value};'`, you write `'UPDATE products SET field= ? ;'`, and can then pass in an "options" object, which is an array of the values you are adding (it plugs them in in order), so for example it can be `cnxion.query('UPDATE products SET field1= ?, field2= ?;'` , [value1, value2], (err) => etc.`